charts/tabulation/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: tabulation
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
Curious where 1.16.0 is coming from here? Since our PHDI version is on `1.0.9`.

Reply:
Oh interesting, I'll change it